plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -370,6 +399,13 @@ class MultiContainerNetworkingFormSection extends mixin(StoreMixin) {
       </span>
     );
 
+    const serviceEndpointsTooltipContent = (
+      <a href="https://docs.mesosphere.com/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/"
```

Comment:
Same here: We do have a `MetadataStore.buildDocsURI` function to build doc links with the correct version!